client/src/app/components/signup-form/signup-form.component.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../styles.scss";
+
+.container {
+  padding: 5em;
+  width: 530px;
+
+  .signup-title {
+    font-size: 48px;
+    margin-bottom: 1em;;
+  }
+
+  .mat-flat-button {
+    background-color: $action-color;
+    color: black;
+    font-size: 16px;
+    height: 42px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    margin-left: 5px;
+  }
+}
+
+@media only screen and (max-width: 600px) {
+  .container {
+    width: initial;
+    padding: 16px;
+
+    .signup-title {
+      text-align: center;
+      margin-bottom: 16px;
+      font-size: 30px;
+      color: $mt-white;
+    }
+  }
+}
```

Comment:
adding these stylings will override the default styling of mat-form-field outline, which can be a solution for removing the extra blue line in the form. 
```
::ng-deep .mat-form-field-appearance-outline .mat-form-field-outline {
      background-color: $mt-white;
      border-radius: 5px;
    }
```
I myself had many problems with the styling of mat-input elements, I looked on StackOverflow and other websites to find solutions and I saw that many people had different problems using it, the default styling is nice but I found it very hard to customize!
I also see that if we use the autocomplete feature of chrome (when you already saved your name and put it in the filed by Chrome), it shows a separate, smaller, weird box inside the input box, so maybe we should use regular input box in the end if we had so many problems with mat-input.

Reply:
@Alessandro100

Reply:
we can override the styling for all mat-input without using ng:deep ... i ll make the change as soon as possible

Reply:
ok so @EspressoCode I looked at the code and the lines you highlighted are not related to Angular Material components. In terms of overriding the style for the mat-input, I read the docs and it seems like the best way to style this is with ng-deep. @marwan011 if you find a more elegant solution, we can go over it together, but we want the finish that angular material brings (animation when clicking, color when focus, etc)